soc/nordic/nrf53/Kconfig

```diff
@@ -177,8 +177,7 @@ config SOC_NRF53_CPUNET_MGMT
 
 config SOC_NRF53_CPUNET_ENABLE
 	bool "NRF53 Network MCU is enabled at boot time"
-	depends on !BT
-	default y if NRF_802154_SER_HOST
+	default y if (BT || NRF_802154_SER_HOST)
```

Comment:
An example where we require 53 cpunet on boot with bluetooth - a multiprotocol application where I want to run another protocol on the cpunet before enabling bluetooth.

Also I do not believe bluetooth driver open/close should control cpunet state and if either of these should change it should be removed from this, rather than on boot. (However, this will require more work and for now I suggest keeping it how it was, with boot on init and on bt driver open)

Reply:
then, why doesn't this code request cpunet as BT code does? As I understand it, cpunet just needs to be booted once by the first requester, turned off by the last releaser. This is why the soc API is ref counted.

Reply:
also, what will happen once this deprecated option is removed?

Reply:
`SOC_NRF53_CPUNET_ENABLE` is not being deprecated is it, so this can be used

Reply:
@gmarull can we proceed this PR? waiting for it integrated to 2.8